packages/suite/src/storage/index.ts

```diff
@@ -4,7 +4,7 @@ import { migrate } from './migrations';
 
 import type { SuiteDBSchema } from './definitions';
 
-const VERSION = 47; // don't forget to add migration and CHANGELOG when changing versions!
+const VERSION = 46; // don't forget to add migration and CHANGELOG when changing versions!
```

Comment:
I assume 46 and 47 haven't been released yet?

Reply:
Yes, it wasnt, i bumped it down again